src/app/admin/components/manage-blog/manage-blog.component.ts

```diff
@@ -132,19 +133,18 @@ export class ManageBlogComponent implements OnInit {
     return index;
   }
 
-  // @Idrice wrong comment?
   // Save to data base this user information
   doSubscription() {
 
-    // @Idrice is that new one??  can you explain this line
     const currentAppBlog: Topic = {
       title: this.title.value,
       content: this.content.value,
       imageUrl: this.imageUrl.value,
       summary: this.summary.value,
       category: this.category.value,
       verified: false,
-      createdDate: `${Date.now()}`, //  Set the createdDate value
+      createdDate: this.now, //  Set the createdDate value
```

Comment:
Comments is better like  set the actual time!!,  to avoid redundance

Reply:
done also